apps/docs/content/components/pickers/ComboBox.mdx

```diff
@@ -1,49 +1,204 @@
 
-## Guidelines
+## Anatomy
 
-TODO: If we have some guidelines about this component's usage
+<FeatureFlag flag="next">
+    ## Guidelines
 
-### Accessibility ?
+    TODO: If we have some guidelines about this component&apos;s usage
 
-TODO: If we have some guidelines about this component and accessibility
+    ### Accessibility?
 
-## Anatomy
+    TODO: If we have some guidelines about this component and accessibility
+</FeatureFlag>
 
-TODO: We have anatomy screenshots from the Figma, we could most likely use them here
+<FeatureFlag flag="rc">
 
-### Concepts
+    TODO: We have anatomy screenshots from the Figma, we could most likely use them here
 
-TODO: links to related concepts
+    ### Concepts
+
+    TODO: links to related concepts
+</FeatureFlag>
 
 ### Composed Components
 
-TODO: links to related components (most likely all component which are used by the slots)
+#### ComboBox
+
+- [Label](./Label)
+- [Helper Message](./HelperMessage)
+- [Error Message](./ErrorMessage)
+- [Header](./Select#header)
+- [Section](./Select#section)
+
+#### ComboBox.Option
+
+- [Text](./Text)
+- [Avatar](Avatar)
+- [Icon](./Icon)
+- [Badge](./Badge)
+
+## ComboBox Examples
+
+### Disabled
+A combo box in a disabled state shows that it exists, but is not available in that circumstance. This can be used to maintain layout continuity and communicate that a field may become available later.
+
+<Example src="ComboBox/docs/disabled" />
+
+### Error
+A combo box can be displayed in an error state to indicate that the selection is invalid.
+
+<Example src="ComboBox/docs/error" />
+
+### Sizes
+A combo box has multiple sizes to choose from.
+The combo box option also changes size based on the size of the combo box.
+
+<Example src="ComboBox/docs/sizes" />
+
+### Labeling
+If a visible label isn't specified, an `aria-label` must be provided to the select for accessibility. If the field is labeled by a separate element, an `aria-labelledby` prop must be provided using the ID of the labeling element instead.
+
+<Example src="ComboBox/docs/labeling" />
+
+### Icon Prefix
+An icon can be displayed at the start of the select trigger.
+
+<Example src="ComboBox/docs/iconPrefix" />
+
+### Text Prefix
+A short text can be displayed at the start of the select trigger.
+
+<Example src="ComboBox/docs/textPrefix" />
+
+### Fluid
+A combo box can take the width of its container.
+
+<Example src="ComboBox/docs/fluid" />
+
+### Controlled
+A combo box can have a controlled selected value. In this example, it shows how it is possible to deselect an option.
+
+<Example src="ComboBox/docs/controlled" />
+
+### Custom Filtering
+A combo box can overridde the default [useFilter](https://react-spectrum.adobe.com/react-aria/useFilter.html) function from RAC. This can be overridden using the `defaultFilter` prop, or by using the items prop to control the filtered list. When items is provided rather than `defaultItems`, ComboBox does no filtering of its own.
```

Comment:
```suggestion
A combo box can overridde the default [useFilter](https://react-spectrum.adobe.com/react-aria/useFilter.html) function from RAC. This can be overridden using the `defaultFilter` prop, or by using the `items` prop to control the filtered list. When `items` is provided rather than `defaultItems`, ComboBox does no filtering of its own.
```